packages/database/test/helpers/syncPointSpec.json

```diff
@@ -3729,7 +3729,7 @@
         "data": {
           "4": { "age": 41, "highscore": 400, "name": "old mama"},
           "5": { "age": 18, "highscore": 1200, "name": "young mama"},
-          "6": { "age": 20, "highscore": 1003, "name": "micheal blub"},
+          "6": { "age": 20, "highscore": 1003, "name": "michael blub"},
```

Comment:
I can't really tell what's going on here. It probably doesn't matter at all, but it's pretty odd to write `micheal` and google search for "micheal blub" (when I beg it to let me search for it) gives me the cute "go fishing" animation.

Happy to drop.